### DISCUSSION TITLE
tables in pdf are cut off

### DISCUSSION THREAD

**Discussion by davehouser1** [HollowMan6/mdbook-pdf#25] [Q&A] [answered]
Hi.
I was able to install rust,mdbooks,mdbooks-pdf, and google-chrome-stable.
I was then adjusted my book.toml to include 

```
[output.html]
[output.pdf]
```

This generates the `output.pdf` file.
Whoever our markdown has tables. When looking in the PDF, all the tables appear to be screen shots / images of the tables. There are two problems with this.

1. the tables are cut off based on the length of the pdf page size. 
2. The tables appear to be screen shots, as I can see slider bars in the images. Obviously these are not usable or needed as this is a pdf.

Is there a way to fix this? Or is this  a known issue?

**Comment by HollowMan6** [comment, marked as answer]
Unfortunately, everything related to how PDFs are rendered is directly controlled by Chrome. We can't do anything on our side. It will be the same result if you directly print the web page into PDF using Chrome.

Hopefully, one mitigation can be setting the scale to a smaller value to make the pdf page fit the table size https://github.com/HollowMan6/mdbook-pdf/blob/main/test_doc/book.toml#L25
